### PR TITLE
More 3.5 changes

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

File: tests/test_inside_docker.sh [hunk 1]
```diff
@@ -20,7 +19,7 @@ yum -y install yum-plugin-priorities yum-utils /bin/mount patch
 pushd tarball-client
 args=(--osgver ${OSG_VERSION} --dver el${OS_VERSION} --basearch x86_64 --bundle osg-wn-client-${OSG_VERSION})
 if [[ $OSG_VERSION == 3.5 ]]; then
-    args+=(--no-prerelease)
+    args+=(--version=3.5.99)
```
Comment (brianhlin) [inline tests/test_inside_docker.sh:22]: What's with the goofy version number?

Comment (matyasselmeci) [reply tests/test_inside_docker.sh:22]: You have to specify _something_. I think it's useful to have something fake to distinguish it from a release.

**Review by brianhlin — APPROVED** [review]
LGTM

File: tests/test_inside_docker.sh (hunk 1 above)
Comment (edquist) [reply tests/test_inside_docker.sh:22]: OSG Release 3.5.100 may come sooner than we expect!

**Pull request merged**